### PR TITLE
Activate tab menu

### PR DESCRIPTION
**Pull request by gyuta**

### CODE REVIEW

File: front/js/components/result.js [hunk 1]
```diff
@@ -32,4 +34,37 @@ const ScoreView = (score) => {
...
+            h("li", {class: state.data3 ? "active" : ""}, [
+              h("a", {onClick: switchToData3}, "data3")
+            ])
+        ])
+      ])
+    )
+}
+
+
+const switchToData1 = (state) => ({...state, data1: true, data2: false, data3: false})
```
Comment (mu2chn) [inline front/js/components/result.js]: data1, data2, data3より`int`とかのほうがいい
```
tabNum: 2
```
みたいな

3つおんなじ様な関数作るんじゃなくて一つにまとめよう

**Review by mu2chn — COMMENTED** [review]
state使うならちゃんと`app()`で初期化して

**Comment by gyuta** [comment]
@face0u0 
view.py の下の方の部分で aggregate['scores'] を計算してるあたりでエラー出て進まんからコメントアウトしたんやが、ちゃんと動く？

**Comment by mu2chn** [comment]
> @face0u0
> view.py の下の方の部分で aggregate['scores'] を計算してるあたりでエラー出て進まんからコメントアウトしたんやが、ちゃんと動く？

masterの最新ブランチ、手元では動いてるけどなあ

なんてエラー？

**Comment by gyuta** [comment]
> 127.0.0.1 - - [06/Mar/2020 19:53:08] "GET /eval?lat=35.041029&lng=135.782641 HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2463, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2449, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1866, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/goto/develop/evalhome/apps/views.py", line 25, in evaluate
    aggregate = getAggregate(scores)
  File "/Users/goto/develop/evalhome/apps/views.py", line 96, in getAggregate
    "count": "hogehoge"
  File "/usr/local/lib/python3.7/site-packages/pymongo/collection.py", line 2380, in aggregate
    **kwargs)
  File "/usr/local/lib/python3.7/site-packages/pymongo/collection.py", line 2299, in _aggregate
    retryable=not cmd._performs_write)
  File "/usr/local/lib/python3.7/site-packages/pymongo/mongo_client.py", line 1464, in _retryable_read
    return func(session, server, sock_info, slave_ok)
  File "/usr/local/lib/python3.7/site-packages/pymongo/aggregation.py", line 148, in get_cursor
    user_fields=self._user_fields)
  File "/usr/local/lib/python3.7/site-packages/pymongo/pool.py", line 613, in command
    user_fields=user_fields)
  File "/usr/local/lib/python3.7/site-packages/pymongo/network.py", line 167, in command
    parse_write_concern_error=parse_write_concern_error)
  File "/usr/local/lib/python3.7/site-packages/pymongo/helpers.py", line 159, in _check_command_response
    raise OperationFailure(msg % errmsg, code, response)
pymongo.errors.OperationFailure: Failed to parse number '' in $convert with no onError value: No digits

@face0u0 
ボタン押した時にサーバ側でこんな感じのエラーがはかれる
一応seeduserは実行した

**Comment by mu2chn** [comment]
> @face0u0
ボタン押した時にサーバ側でこんな感じのエラーがはかれる
一応seeduserは実行した

@gyuta 
mongodbのバージョン教えて

**Comment by gyuta** [comment]
@face0u0 
MongoDB shell version v4.2.2

**Comment by gyuta** [comment]
@face0u0 
`point: 5000` のレコードが `range: ""` で換算されて、それを int に直す時にエラーでてたから修正したよ

**Pull request closed without merging**